src/Delegate/NotFoundDelegate.php

```diff
@@ -78,7 +87,7 @@ private function generatePlainTextResponse(ServerRequestInterface $request)
             ->write(sprintf(
                 'Cannot %s %s',
                 $request->getMethod(),
-                (string) $request->getUri()
+                (string)$request->getUri()
```

Comment:
Revert this change please. We don;t have currently sniff to check it, so all seems to be fine, but for consistency we should have here single space.